zowex/src/main.rs

```diff
@@ -129,59 +132,40 @@ fn main() -> std::io::Result<()> {
     }
 }
 
-/**
- * Convert a vector of command line arguments into a single string of arguments.
- * @param cmd_line_args
- *      The user-supplied command line arguments to the zowe command.
- *      Each argument is in its own vector element.
- * @returns
- *      A String containing all of the command line arguments.
- */
-fn arg_vec_to_string(arg_vec: Vec<String>) -> String {
```

Comment:
The arg_vec_to_string function is now used in a second, unrelated place in the next-daemon-enable branch. So, it will be resurrected when that branch merges.